fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -521,6 +521,7 @@ private Query getFilterQuery(Set<ApiFilter> filters, int perPage) {
             filterQueryBuilder.add(new BooleanClause(new MatchAllDocsQuery(), BooleanClause.Occur.MUST));
         }
 
+        LOG.debug("Filter Query: {}", filterQueryBuilder.build().toString());
```

Comment:
This will result in the build being done twice every time.  
Perhaps this is not the outcome you want.